packages/core/webpack/config/loaders.js

```diff
@@ -31,47 +31,49 @@ module.exports = props => {
   }, props);
 
   const babelPresets = [
-    ["@babel/preset-env", {
+    [require.resolve("@babel/preset-env"), {
       modules: false,
       forceAllTransforms: true
     }],
-    "@babel/preset-react"
+    require.resolve("@babel/preset-react")
   ];
 
   const babelPlugins = [
 
     // Stage 0
-    "@babel/plugin-proposal-function-bind",
+    require.resolve("@babel/plugin-proposal-function-bind"),
 
     // Stage 1
-    "@babel/plugin-proposal-export-default-from",
-    "@babel/plugin-proposal-logical-assignment-operators",
-    ["@babel/plugin-proposal-optional-chaining", {loose: false}],
-    ["@babel/plugin-proposal-pipeline-operator", {proposal: "minimal"}],
-    ["@babel/plugin-proposal-nullish-coalescing-operator", {loose: false}],
-    "@babel/plugin-proposal-do-expressions",
+    require.resolve("@babel/plugin-proposal-export-default-from"),
+    require.resolve("@babel/plugin-proposal-logical-assignment-operators"),
+    [require.resolve("@babel/plugin-proposal-optional-chaining"), {loose: false}],
+    [require.resolve("@babel/plugin-proposal-pipeline-operator"), {proposal: "minimal"}],
+    [require.resolve("@babel/plugin-proposal-nullish-coalescing-operator"), {loose: false}],
+    require.resolve("@babel/plugin-proposal-do-expressions"),
 
     // Stage 2
-    ["@babel/plugin-proposal-decorators", {legacy: true}],
-    "@babel/plugin-proposal-function-sent",
-    "@babel/plugin-proposal-export-namespace-from",
-    "@babel/plugin-proposal-numeric-separator",
-    "@babel/plugin-proposal-throw-expressions",
+    [require.resolve("@babel/plugin-proposal-decorators"), {legacy: true}],
+    require.resolve("@babel/plugin-proposal-function-sent"),
+    require.resolve("@babel/plugin-proposal-export-namespace-from"),
+    require.resolve("@babel/plugin-proposal-numeric-separator"),
+    require.resolve("@babel/plugin-proposal-throw-expressions"),
 
     // Stage 3
-    "@babel/plugin-syntax-dynamic-import",
-    "@babel/plugin-syntax-import-meta",
-    ["@babel/plugin-proposal-class-properties", {loose: false}],
-    "@babel/plugin-proposal-json-strings",
+    require.resolve("@babel/plugin-syntax-dynamic-import"),
+    require.resolve("@babel/plugin-syntax-import-meta"),
+    [require.resolve("@babel/plugin-proposal-class-properties"), {loose: false}],
+    require.resolve("@babel/plugin-proposal-json-strings"),
 
     // React specific
-    "@babel/plugin-transform-react-constant-elements"
+    require.resolve("@babel/plugin-transform-react-constant-elements")
 
   ];
 
   if (process.env.NODE_ENV === "development") {
-    babelPlugins.push("react-hot-loader/babel");
-    babelPlugins.push("@babel/plugin-transform-react-inline-elements");
+    babelPlugins.push(
+      require.resolve("react-hot-loader/babel"),
+      require.resolve("@babel/plugin-transform-react-inline-elements")
```

Comment:
I'm totally fine with adding in these `require.resolve` wraps, but just out of curiousity, what is this protecting against?

Reply:
Is because of the monorepo
Package managers install the dependencies in their own package/folders/node_modules, but then the crosslinking makes the resolution relative to the node_modules in the current working directory. This part is not subject to the webpack.resolve configuration, so this is the most reliable way I found that doesn't interfere when installing as a package.